README.md

```diff
@@ -29,14 +31,23 @@ Then you can install in this environment `xeus-cling` and its dependencies
 conda install xeus-cling -c conda-forge
 ```
 
-Or you can install it directly from the sources, if all the dependencies are already installed.
+### Installation from source
+
+You will first need to create a new environment and install the dependencies:
+
+```bash
+conda create -n xeus-cling -c conda-forge cmake xeus=0.23.3 cling=0.6.0 clangdev=5.0 llvmdev=5 nlohmann_json cppzmq=4.3.0 xtl=0.6.9 pugixml cxxopts=2.1.1
+conda activate xeus-cling
+```
+
+You can then compile the sources:
 
 ```bash
-cmake -DCMAKE_INSTALL_PREFIX=your_conda_path -DCMAKE_INSTALL_LIBDIR=your_conda_path/lib
+cmake -D CMAKE_INSTALL_PREFIX=${CONDA_PREFIX} -D CMAKE_C_COMPILER=$CC -D CMAKE_CXX_COMPILER=$CXX -D CMAKE_INSTALL_LIBDIR=${CONDA_PREFIX}/lib -D DOWNLOAD_GTEST=ON
```

Comment:
I think that $CXX and $CC are environment variables available in the CI only. They should not be specified here.

Reply:
They also seem to be available in a local conda environment:

```
$ echo $CC
/home/jtp/miniconda/envs/xeus-cling/bin/x86_64-conda_cos6-linux-gnu-cc
```

Reply:
OK looks like they are set by some of the dependencies (I would bet on llvm or clang or cling), because they are not available in an environment without these packages.

Anyway, let's ignore my comment ;)